provisioning/device/tests/AuthenticationProviderX509Tests.cs

```diff
@@ -20,5 +25,23 @@ public void AuthenticationProviderX509_ThrowsWhenMissingCert()
             // assert
             act.Should().Throw<ArgumentException>();
         }
+
+        [TestMethod]
+        public void AuthenticationProviderX509_Works()
+        {
+            // arrange
```

Comment:
We can leave an extra line below this comment if we have a code block instead of a single line in this section.